src/Microsoft.Diagnostics.Monitoring/DiagnosticServices.cs

```diff
@@ -10,11 +10,12 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
+using Graphs;
 using Microsoft.Diagnostics.Monitoring.Logging;
 using Microsoft.Diagnostics.NETCore.Client;
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using static System.FormattableString;
```

Comment:
I don't think I've seen this Invariant pattern in any codebase I've looked at. Why are we doing this?

Reply:
I'll revert it.